terraform/deployments/cluster-infrastructure/variables.tf

```diff
@@ -147,6 +147,30 @@ variable "grafana_db_seconds_until_auto_pause" {
   default     = 7200
 }
 
+variable "chat_db_min_capacity" {
+  type        = number
+  description = "Minimum capacity of the Grafana RDS Aurora Serverless database."
```

Comment:
A few references to Grafana lingering in this file.

Reply:
Fixed